### ISSUE TITLE
NTR: Peripheral nerve biopsy: giant axons (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9280]
New term request Peripheral nerve biopsy: giant axons (segal_120905205838): Axonal swelling

**Comment by pnrobinson** [comment]
@MickeySegal are you referring to the phenotype in figures 4/5 https://jnnp.bmj.com/content/76/6/825

Proposed term label
Peripheral axonal distension
Def: Abnormal increase in the cross-section area of peripheral nerve axons (up to 20-25 micrometers in diameter) .

Comment; In giant axonal neuropathy, distended axons are observed in conjunction with thin myelin sheaths. Distended axons may be filled with masses of neurofilaments. See Figures 4 and 5 of PMID:15897506

**Comment by MickeySegal** [comment]
Yes, our term is based on giant axonal neuropathy.  Your wording is fine.

**Comment by pnrobinson** [comment]
@MickeySegal 
I am reviewing this today and am not sure whether this is actually a synonym of 
[Enlarged peripheral nerve HP:0012645](https://hpo.jax.org/app/browse/term/HP:0012645)?
This term is used current to annotate Leprosy (which can have Peripheral nerve thickening ) and a form of Lipodystrophy (about which I cannot find information about the enlarged nerves) and a third entry from Orphanet that seems erroneous.



https://hpo.jax.org/app/browse/term/HP:0012645

**Comment by MickeySegal** [comment]
In many instances such as this we draw the distinction between a palpable macroscopic finding and a microscopic finding of individual elements.  As an example here, a nerve may be enlarged because of excessive myelin with normal axonal size (HP:0030173, HP:0003383).  So it makes sense to have the "Peripheral axonal distension" term in addition to the "Enlarged peripheral nerve" term.

**Comment by pnrobinson** [comment]
@MickeySegal 
good point, I completely agree!

I have added this as
HP:0034711
Peripheral axonal distension

and I updated the definition slightly: An abnormal histological finding characterized by an increase in the cross-section area of peripheral nerve axons (up to 20-25 micrometers in diameter).
